### ISSUE TITLE
Any rpc/action which handles the state data

### ISSUE THREAD

**Issue by surabhi-Siemens** [sysrepo/sysrepo#2912]
Assuming that streams are defined and handled like state data:

1.)	They are not influence or part of the configuration datastore and thus not influenced by any lock to a datastore.
2.)	They can only read by using the <get> command.
3.)	An action / rpc is able to “Netconf servers must serialize;” consistently update stream entries if they are defined “read-only”/state data

Any <rpc>/<action> which handles state data should be able to consistently update this data!

**Comment by michalvasko** [comment]
What streams are you talking about?

**Comment by surabhi-Siemens** [comment]
Let me rephrase the question, via user-rpc call we are able to modify config data, suppose if there is and sate node, is it possible to update it by RPC call?

**Comment by michalvasko** [comment]
There is no standardized RPC to modify state (operational) data. However, *sysrepo* allows that in several ways so defining your own RPC that would use some of these methods is always possible. In general, these data should be provided by system apps/daemons as they should inform about their state.

**Comment by surabhi-Siemens** [comment]
Any example where i can find these methods?

**Comment by michalvasko** [comment]
Everything is in the [docs](https://netopeer.liberouter.org/doc/sysrepo/devel/html/index.html#oper_data).
